iOS/Login/LoginTests.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  LoginTest.swift
+//  MobileSDKUITest
+//
+//  Created by Brandon Page on 2/2/18.
+//
+
+import XCTest
+
+class LoginTests: XCTestCase {
+    private var username = "circleci@mobilesdk.com"
+    private var password = "test1234"
+    private var appLoadError = "App did not load."
+    private var timeout:double_t = 30
+    
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    func testLogin() {
+        let app = TestApplication()
+        let loginPage = LoginPageObject(testApp: app)
+        let authPage = AuthorizationPageObject(testApp: app)
+        
+        app.launch()
+        loginPage.setUsername(name: username)
+        loginPage.setPassword(password: password)
+        loginPage.tapLogin()
+        authPage.tapAllow()
+        
+        // Assert App loads
+        switch app.appType {
+        case .nativeObjC, .nativeSwift:
+            XCTAssert(app.navigationBars["Mobile SDK Sample App"].waitForExistence(timeout: timeout), appLoadError)
+        case .hybridLocal, .hyrbidRemote:
+            let titleText = (app.appType == .hybridLocal) ? "Users" : "Salesforce Mobile SDK Test"
```

Comment:
Should we change the hybrid app templates so that they use the same text in the navigation bar than the native apps?